src/game/gui/text/enums.h

```diff
@@ -0,0 +1,43 @@
+#ifndef TEXT_ENUMS_H
+#define TEXT_ENUMS_H
+
+#include <stdint.h>
+
+typedef enum text_vertical_align
+{
+    ALIGN_TEXT_TOP,
+    ALIGN_TEXT_MIDDLE,
+    ALIGN_TEXT_BOTTOM
+} text_vertical_align;
+
+typedef enum text_horizontal_align
+{
+    ALIGN_TEXT_LEFT,
+    ALIGN_TEXT_CENTER,
+    ALIGN_TEXT_RIGHT
```

Comment:
```suggestion
    TEXT_ALIGN_LEFT,
    TEXT_ALIGN_CENTER,
    TEXT_ALIGN_RIGHT
```
& same for text_vertical_align.

Reply:
Probably sounds nicer. Done.